nemo/src/io/parser.rs

```diff
@@ -326,6 +327,30 @@ impl<'a> RuleParser<'a> {
         traced("parse_close_parenthesis", space_delimited_token(")"))
     }
 
+    /// Matches an opening parenthesis,
+    /// then gets an object from the parser,
+    /// and finally matches an closing parenthesis.
+    pub fn in_parenthesis<'b, O, F>(
+        &'a self,
+        parser: F,
+    ) -> impl FnMut(Span<'a>) -> IntermediateResult<O>
+    where
+        O: Debug + 'a,
+        F: FnMut(LocatedSpan<&'a str>) -> IntermediateResult<O> + 'a,
+    {
+        traced(
+            "in_parenthesis",
+            map_error(
+                delimited(
+                    delimited(multispace_or_comment0, token("("), multispace_or_comment0),
```

Comment:
could actually be `space_delimited_token("(")`, and that is precisely `RuleParser::parse_open_parenthesis`.